src/pages/[project]/data/index.js

```diff
@@ -1,7 +1,11 @@
 import { Component } from "react";
 import { Layout } from "@/layouts/Layout";
 import DataForm from "@/components/DataForm";
-import { attributes, react as DataContent } from "@/content/pages/data.md"
+import { attributes, react as DataContent } from "@/content/pages/mmp-data.md";
+import {
+  attributes as lampAttributes,
```

Comment:
I was curious if there was a reason the lamp implementation hasn't been included below yet compared to src/pages/[proejct]/index.js

Reply:
Oh gotcha, I know you mentioned that yesterday i just wanted to double check, thanks Sam